rtris.py

```diff
@@ -33,10 +33,15 @@
 		conf=json.load(conffile)
 		strg=conf["strg"]
 try:
-	update=conf["update"]
+	tmp=conf["update"]
```

Comment:
I want to avoid using unecessary variables, so please replace `tmp` with `update`